pythonforandroid/toolchain.py

```diff
@@ -69,7 +78,7 @@ def format(self, record):
                                     # handler and reset the level
     logger.setLevel(logging.INFO)
     logger.touched = True
-    ch = logging.StreamHandler(stdout)
+    ch = logging.StreamHandler(stdout) if sys.stdout.isatty() else logging.NullHandler()
```

Comment:
Why use a NullHandler if the stdout isn't a tty? Don't we still want to send the output?